### ISSUE TITLE
Unuseful dependencies

### ISSUE THREAD

**Issue by c3s4** [mjmlio/json2mjml#3]
Hi, it seems that dependencies are not used at all.
Is possible to remove them?

I noticed this because installing this lib cause the installation of a lot of unused packages.

**Comment by ngarnier** [comment]
Care to be more specific and explain which dependencies you are referring to, and how you know they are not used?

Creating an issue with so little detail doesn't make things easy for us.

**Comment by c3s4** [comment]
Hi sorry for the lack of details, I mean _all_ dependencies (not the dev ones).
I think that are not used because inside the two source files they are never referenced. The only one referenced is the `commander`, but you are using the `babel-cli` so it will available after the build phase. Maybe the `commander` is right to have as dependency.

I cloned this project, removed from package.json then executed `yarn test` and the test has been passed, so I think these dependencies are not really needed. To check the cli I tried to use it after the build and everything works as expected.

Just to be more clear these are the dependencies I'm referring to:
```
"commander": "^2.11.0", (maybe this not)
"htmlparser2": "^3.9.2",
"lodash": "^4.17.2",
"mjml": "^3.3.5"
```

I'm telling this mainly because the `mjml` one: it cause a lot of unused packages download and it is referencing an old version of mjml.

Thank you
